src/style/font.ts

```diff
@@ -0,0 +1,29 @@
+import { ThemeCSSRule } from '.'
+
+export interface FontTheme {
+  /** Defaults to sans-serif */
```

Comment:
I'm using these mostly to avoid errors, TypeScript or ESLint will not complain if we have
```css
  font-family: "Open San", sans-serif; /* Note the typo on Sans */
```
So whenever we need to refer to font family I'm using the values from the theme. Another feature of these is that by default headers are going to use `font.accent`, and the page is going to use `font.sans` for text. These values are easy to overwrite using `font.text`/`font.h1h6`.

Reply:
I agree about having defined constants to mean font names to avoid spellings.  I just don't think that all websites will follow the scheme `sans` / `serif` / `mono` / `accent` etc ...

Instead, let's just call it `Record<string, string>` and let the websites subclass what `font` needs to be.